webrtc-android-sample-app/src/main/java/io/antmedia/webrtc_android_sample_app/ScreenCaptureActivity.java

```diff
@@ -110,7 +103,21 @@ else if(checkedId == R.id.rbRear) {
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data)
     {
-        webRTCClient.onActivityResult(requestCode, resultCode, data);
+        // If the device version is v29 or higher, screen sharing will work service due to media projection policy.
+        // Otherwise media projection will work without service
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q){
+            MediaProjectionService service = new MediaProjectionService();
+
+            service.setListener(mediaProjection -> {
+                webRTCClient.setMediaProjection(mediaProjection);
+                webRTCClient.onActivityResult(requestCode, resultCode, data);
+            });
+
+            service.start(getApplicationContext(), data);
```

Comment:
As far as I remember starting service is not working in this way. 
https://developer.android.com/guide/components/services#StartingAService

Reply:
I confirm that the current code works fine. In the start function, Context.startForegroundService is used instead of startService because of that when sharing the screen, the application has to run in the background so we should use startForegroundService 

https://developer.android.com/reference/android/content/Context#startForegroundService(android.content.Intent)

https://github.com/ant-media/WebRTC-Android-SDK/pull/1/files#diff-1e7c6406564e3187def12a792db66f370b1dbe3f7b90bf498850973ed0795cf8R41

Reply:
OK. Merging.